ush/python/pygw/src/schema_pkg/check_schema.py

```diff
@@ -0,0 +1,98 @@
+import logging
```

Comment:
I am not quite clear on the need of `build_schema.py` and `check_schema.py`. 
They are convenient, but I am not sold on their need yet.
I'll think about it some more.